README-Off-Season.md

```diff
@@ -77,7 +77,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [GE Aerospace](https://jobs.gecareers.com/aviation/global/en/job/R3718914/GE-Aerospace-US-Digital-Technology-Coop-Spring-2024) | Cincinnati, OH <br/> Van Buren Township, MI | Spring 2024 | GE Aerospace US Digital Technology Coop (No Sponsorship) |
 | [Sony Pictures](https://www.sonypicturesjobs.com/job/culver-city/software-engineer-golang-intern-fall-2023/22978/51635682320) | Culver City, CA | Fall 2023 | Software Engineer Golang Intern |
 | [SRC Inc.](https://jobs.jobvite.com/src-inc/job/oHynnfwJ) | Syracuse, NY <br/> Herndon, VA <br/> San Antonio, TX | Fall 2023 | Software Engineering Intern |
-| [Parallel Systems](https://moveparallel.com/careers/?gh_jid=4026759007) | Los Angeles, CA | Fall 2023 | Software Engineer Intern | 
+| [Parallel Systems](https://moveparallel.com/careers/?gh_jid=4026759007) | Los Angeles, CA | Fall 2023 | Software Engineer Intern(US Citizenship Required) | 
```

Comment:
I might be blind, but where did it say that US citizenship was required for Parallel Systems? Did you receive an email about it or something?